pkg/kubelet/container/runtime.go

```diff
@@ -84,6 +84,8 @@ type Runtime interface {
 	ContainerCommandRunner
 	// ContainerAttach encapsulates the attaching to containers for testability
 	ContainerAttacher
+	// Get Container NetNamespacePath
+	GetNetNs(containerId string) (string, error)
```

Comment:
> The eventual destination is that the CNI logic lives entirely withing the docker runtime, right?

I didn't necessarily get that conclusion from the whole libnetwork remote driver PR though.  What I got was "we'll continue to prototype but if libnetwork keeps making it really hard we'll just use CNI + Kubernetes".  Does that match your expectation?